src/ggrc/models/person_profile.py

```diff
@@ -8,29 +8,28 @@
 from sqlalchemy.orm import relationship
 
 from ggrc import db
-from ggrc.models.inflector import ModelInflectorDescriptor
-from ggrc.models.mixins.base import Identifiable
+from ggrc.models.mixins.base import Base
 
 # Offset for default last seen what's new date in days
 DEFAULT_LAST_SEEN_OFFSET = 60
 
 
 def default_last_seen_date():
   """Function counts last_seen_whats_new date for new users"""
-  return datetime.now() - timedelta(DEFAULT_LAST_SEEN_OFFSET)
+  default_date = datetime.now() - timedelta(DEFAULT_LAST_SEEN_OFFSET)
+  return default_date.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
please return datetime here, but with empty microseconds. Smth like:
now = datetime.datetime.now() 
now.replace(microsecond=0) 
now - timedelta(DEFAULT_LAST_SEEN_OFFSET)

this will grant you that every time you will have here datetime instance.